documentation/src/docs/reader-group-design.md

```diff
@@ -9,85 +9,90 @@ You may obtain a copy of the License at
 -->
 # Reader Groups Design
 
-_To be updated to reflect recent implementation details (05/07/2017)_
+## Motivation
+A set of Readers can be grouped together in order that the set of Events in a Stream can be read in parallel. This grouping of Readers is called a Reader Group. Pravega guarantees that each Event in the Stream is read by exactly one Reader in the Reader Group.
 
-## Motivation 
-A set of Readers can be grouped together in order that the set of Events in a Stream can be read in parallel.  The grouping of Readers is called a Readers Group.  Pravega guarantees that each Event in the Stream is read by exactly one Reader in the Readers Group. 
-
-Each Reader in a ReaderGroup is assigned zero or more Segments.
-The Reader assigned to a Segment is the only Reader within the ReaderGroup that reads Events from that Segment.  This is the fundamental mechanism by which Pravega makes ordering guarantees of Event delivery to a Reader – a Reader will receive Events in the order they were published into a Segment.
+Each Reader in a Reader Group is assigned zero or more Segments.
+The Reader assigned to a Segment is the only Reader within the Reader Group that reads Events from that Segment. This is the fundamental mechanism by which Pravega makes ordering guarantees of Event delivery to a Reader – a Reader will receive Events in the order they were published into a Segment.
```

Comment:
I would rather prefer `written` instead of `published` as this is a storage system (different from MOM systems).

Reply:
accepted suggestion